versioned_docs/version-2.11.x/io-debezium-source.md

```diff
@@ -268,6 +268,7 @@ You can use one of the following methods to create a configuration file.
   inputs: [ "debezium-postgres-topic" ]
   archive: "connectors/pulsar-io-debezium-postgres-@pulsar:version@.nar"
   parallelism: 1
+  topicName: "public/default/log-debezium-postgres"
```

Comment:
Do you verify it?
I see the configuration in Pulsar just "log-debezium-postgres" [here](https://github.com/apache/pulsar/blob/master/pulsar-io/debezium/postgres/src/main/resources/debezium-postgres-source-config.yaml#L35).

<img width="731" alt="image" src="https://user-images.githubusercontent.com/55571188/234445547-061ffa96-bcca-4556-8007-8cf4ae57184b.png">

Reply:
It seems the tenant and namespace part can be auto appended so both is correct?